packages/vue/src/components/molecules/SfScrollable/SfScrollable.vue

```diff
@@ -22,18 +22,38 @@ export default {
     Simplebar,
     SfButton,
   },
+  inject: {
```

Comment:
IMO we shouldn't change this component. You can use provide/inject just for depending components. Pleas use props :)

Reply:
now that's clear for me, thx :)